spec/magma_attribute_spec.rb

```diff
@@ -183,7 +183,7 @@
       attribute = Magma::Attribute.new("name", model, {
         validation: { type: "Regexp", value: /^[a-zA-Z]{1}$/ }
       })
-      json_validation_object = attribute.json_template[:validation_object].as_json
+      json_validation_object = attribute.json_template[:validation]
```

Comment:
Despite its clearly inapt name, `#json_template` doesn't actually return json, it just returns an object that can be successfully turned into json - the actual transformation to json happens very late, [here](https://github.com/mountetna/magma/blob/40601560edc2d97dd6ef18880acad21c513e789a/lib/magma/server/retrieve.rb#L117) after the json_template is bundled with a bunch of others - so I think this `attribute.json_template[:validation]` should not yet be a JSON string?

Reply:
That's what I originally had, but it did not get converted to a JSON string at any point before getting called by the retrieve endpoint.